### PR TITLE
[Bulk] Validation Fixes

### PR DESCRIPTION
**Pull request by AqMoh**
This PR fixes 50 issues.

### CODE REVIEW

File: BingMaps/v8-web-control/articles/cross-platform-bing-maps-v8-apps.md [hunk 1]
```diff
@@ -55,14 +56,14 @@ In both of these scenarios, add the following content URI's and set WinRT access
 
 ## Native UWP Apps via Apache Cordova
 
-To whitelist these domains in a native Windows 10, open the **config.xml** file in your Cordova project and near top of the file you should see "&lt;access origin="\*" /&gt;" after this line add the following two lines:
+To safeelist these domains in a native Windows 10, open the **config.xml** file in your Cordova project and near top of the file you should see "&lt;access origin="\*" /&gt;" after this line add the following two lines:
```
Comment (jmart1428) [inline BingMaps/v8-web-control/articles/cross-platform-bing-maps-v8-apps.md:59]: remove extra 'e' in 'safeelist'

File: BingMaps/v8-web-control/articles/cross-platform-bing-maps-v8-apps.md [hunk 2]
```diff
@@ -55,14 +56,14 @@ In both of these scenarios, add the following content URI's and set WinRT access
...
 
 ```xml
 <access origin="*.bing.com" subdomains="true" />
 <access origin="*.virtualearth.net" subdomains="true" />
 ```
 
-If you project doesn’t work after whitelisting these URLs ensure that the version of Windows that your app targets is 10. There is a preference property in the **config.xml** file that specifies the target version of Windows. Make sure it is set to 10.0 like this:
+If you project doesn’t work after safelisting these URLs ensure that the version of Windows that your app targets is 10. There is a preference property in the **config.xml** file that specifies the target version of Windows. Make sure it is set to 10.0 like this:
```
Comment (jmart1428) [inline BingMaps/v8-web-control/articles/cross-platform-bing-maps-v8-apps.md:66]: Just happened to notice typo. "If you project" should be "If your project". Might as well fix if you're already going back into this file.

File: BingMaps/v8-web-control/creating-and-hosting-map-controls/map-control-branches.md [hunk 3]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Map Control Branches | Microsoft Docs"
+description: "This article describes the two branches of the Bing Maps V8 SDK that can be accessed, as well as a URL can be used to load the experimental branch."
```
Comment (jmart1428) [inline BingMaps/v8-web-control/creating-and-hosting-map-controls/map-control-branches.md:3]: "as well as a URL **that** can be used..."

**Comment by AqMoh** [comment]
**Global effort to fix build validation errors**
@stevemunk 
The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you'd like to see. Thank you!

File: BingMaps/v8-web-control/creating-and-hosting-map-controls/setting-map-control-parameters.md [hunk 4]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Setting Map Control Parameters2 | Microsoft Docs"
+description: "This article provides guidelines on how to add a parameter to the map control src URL and localizing the map. A list of parameter URLs with their descriptions are also provided."
```
Comment (stevemunk) [inline BingMaps/v8-web-control/creating-and-hosting-map-controls/setting-map-control-parameters.md:3]:
```suggestion
description: "This article explains how to set map control parameters and includes descriptions for all available parameters, also describes how to localize the map control including overriding the detected settings using the setLang and setMkt parameters."
```

**Review by stevemunk — CHANGES_REQUESTED** [review]

**Comment by stevemunk** [comment]
@AqMoh Please let me know when you are finished making updates, thanks!

**Comment by AqMoh** [comment]
@stevemunk
Had to sort out some merge conflicts, but should be good to go now. Thank you!!

**Comment by jmart1428** [comment]
@stevemunk  Are there any additional changes that you'd like to see in this PR? Thank you for your feedback!

**Pull request merged**